### PR TITLE
Update backup.md

### PR DESCRIPTION
**Pull request by sunwenli**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix anchor link

### CODE REVIEW

**Comment by sunwenli** [comment]
should I remove line 1-6, the machine_translated mark @evillique

**Comment by evillique** [comment]
> should I remove line 1-6, the machine_translated mark @evillique

If you know that everything looks good and there are no translation errors then I think you can :)

**Comment by sunwenli** [comment]
> > should I remove line 1-6, the machine_translated mark @evillique
> 
> If you know that everything looks good and there are no translation errors then I think you can :)

ok

File: docs/zh/operations/backup.md [hunk 1]
```diff
@@ -1,13 +1,7 @@
----
-machine_translated: true
-machine_translated_rev: 72537a2d527c63c07aa5d2361a8829f3895cf2bd
-toc_priority: 49
-toc_title: "\u6570\u636E\u5907\u4EFD"
----
```
Comment (evillique) [inline docs/zh/operations/backup.md:1]:
```suggestion
---
toc_priority: 49
toc_title: "\u6570\u636E\u5907\u4EFD"
---

```

Comment (sunwenli) [reply docs/zh/operations/backup.md:1]: OK

**Pull request merged**